lib/AST/Decl.cpp

```diff
@@ -2180,7 +2180,7 @@ ArrayRef<ProtocolDecl *>
 AbstractTypeParamDecl::getConformingProtocols() const {
   auto *dc = getDeclContext();
   if (!dc->isValidGenericContext())
-    return nullptr;
+    return {};
```

Comment:
When iterating, this gives a nullptr back. I have to change it to `{}` to avoid iterating on a nullptr.